common/Mailer.js

```diff
@@ -2,8 +2,7 @@ const nodemailer = require('nodemailer');
 const mandrillTransport = require('nodemailer-mandrill-transport');
 const emailTemplates = require('email-templates');
 const templatesDir = require('path').resolve(__dirname, '..', 'email-templates');
-
-const credentialMandrill = require('common/makeCredential')('mandrill');
+const credentialMandrill = process.env.KINDLIZED_MANDRILL;
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).